app/Models/User.php

```diff
@@ -40,4 +40,9 @@ class User extends Authenticatable
     protected $casts = [
         'email_verified_at' => 'datetime',
     ];
+
+    public function bots()
+    {
+        return $this->hasMany('App\Models\Bot');
```

Comment:
$this->hasMany('App\Models\Bot');

chuyển thành: $this->hasMany(Bot::class); cho gọn tương tự các models khác